src/app/armory/__snapshots__/weapon-roll-export.test.ts.snap

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing
+
+exports[`formatWeaponRollForExport should format a basic weapon roll correctly 1`] = `
+"[Test Weapon]
```

Comment:
You took the instructions too literally. The name of the weapon should not have square brackets around it.